src/FSharp.Formatting.Razor/Razor.fs

```diff
@@ -25,10 +25,38 @@ open System.Dynamic
 open System.Collections.Generic
 open System.Collections.Concurrent
 open FSharp.Formatting.Common
+
+#if !NETSTANDARD2_0
 open RazorEngine.Text
 open RazorEngine.Templating
 open RazorEngine.Configuration
 open RazorEngine.Compilation.ReferenceResolver
+#endif
+
+//module tst =
```

Comment:
is this needed?

Reply:
You'r right, this part is not compile for netstandard anyway